Automattic/wp-calypso
# Theme Tiers: Fix Sensei setup to work with new Tiers

Related to:
- https://github.com/Automattic/dotcom-forge/issues/6351

## Proposed Changes

* We moved the installation to after the site has gone Atomic.
* We moved the activation of the theme + style variation to after the installation.
* Cleaned up some code.

## Why are these changes being made?


* As part of the Theme Tiers CANI we want to clean up some issues that we've left during the execution of the main theme tiers project. In this case, we had to set the Sensei theme tier to free so that the Sensei onboarding flow worked.

## Testing Instructions


- Apply this patch D153777-code
- Use the Calypso Live link
- Go to https://{live link}/setup/sensei and complete the onboarding flow.
- Make sure it completes without any issue and Course theme is set as the theme of the site.
- Also check that the selected style variation is selected.


## Pre-merge Checklist
